cli/commands/helpers/json_test.go

```diff
@@ -22,3 +27,111 @@ func TestPrintJSON(t *testing.T) {
 	require.NoError(t, PrintJSON(testInput, writer))
 	assert.Equal("{\n  \"commandAnd\": \"echo bar && exit 1\",\n  \"commandLessThan\": \"echo foo >> output.txt\"\n}\n\n", buf.String())
 }
+
+func TestPrintWrappedJSON(t *testing.T) {
+	assert := assert.New(t)
+
+	check := types.FixtureCheckConfig("check")
+	check.Command = "echo foo >> output.txt"
+
+	w := types.Wrapper{
+		Type:  "CheckConfig",
+		Value: check,
+	}
+	output, err := json.Marshal(w)
+	assert.NoError(err)
+
+	buf := new(bytes.Buffer)
+	writer := io.Writer(buf)
+	require.NoError(t, PrintWrappedJSON(check, writer))
+	assert.JSONEq(string(output), buf.String())
+}
+
+func TestPrintWrappedJSONList(t *testing.T) {
+	assert := assert.New(t)
+
+	check1 := types.FixtureCheckConfig("check1")
+	check2 := types.FixtureCheckConfig("check2")
+
+	w1 := types.Wrapper{
+		Type:  "CheckConfig",
+		Value: check1,
+	}
+	w2 := types.Wrapper{
+		Type:  "CheckConfig",
+		Value: check2,
+	}
+	output1, err := json.Marshal(w1)
+	assert.NoError(err)
+	output2, err := json.Marshal(w2)
+	assert.NoError(err)
+
+	buf := new(bytes.Buffer)
+	writer := io.Writer(buf)
+
+	require.NoError(t, PrintWrappedJSONList([]types.Resource{check1, check2}, writer))
+	// trim \n and white space for equal comparison
+	output3 := strings.Replace(buf.String(), "\n", "", -1)
+	output3 = strings.Replace(output3, " ", "", -1)
+	assert.Equal(string(output1)+string(output2), output3)
```

Comment:
This is a bit unsafe because the output might differ. We should use the JSONEq function.

We can do it line by line by using strings.Split.

Reply:
@echlebek JSONEq does not recognize the output of `PrintWrappedJSONList` as valid json:
```
{check1}
{check2}
```
I think we'll have to fudge the expected value in some other way then. In what case would the output differ with all stripped whitespace? I would _expect_ the contents to be the same.

Reply:
Right; that's why you would split the lines and compare each one individually. :)

Reply:
The contents can differ; serializing JSON is not necessarily a deterministic process. (Map iteration order can change things a lot.)

Reply:
How would we split `buf.String()`:
```
{
  "type": "CheckConfig",
  "spec": {
    "command": "command",
    "environment": "default"
  }
}
{
  "type": "CheckConfig",
  "spec": {
    "command": "command",
    "environment": "default"
  }
}
```
So that it is equal to output1 or output2 without trimming whitespace or newline characters:
```
{"type": "CheckConfig","spec": {"command": "command","environment": "default"}}
```

Reply:
A json decoder perhaps?

Reply:
This feels weird but works.
```
	output3 := strings.Split(buf.String(), "}\n{")
	assert.JSONEq(string(output1), output3[0]+"}")
	assert.JSONEq(string(output2), "{"+output3[1])
```

Reply:
Oh, good point, I forgot that it tabs the output.

I like what you have above a bit better. Let's go with that for now!

Reply:
Done!